packages/client/tests/functional/fluent-api/prisma/_schema.ts

```diff
@@ -0,0 +1,74 @@
+import { idForProvider } from '../../_utils/idForProvider'
+import testMatrix from '../_matrix'
+
+export default testMatrix.setupSchema(({ provider }) => {
+  const id = idForProvider(provider)
+  return /* Prisma */ `
+    generator client {
```

Comment:
Wondering if this entire schema is used in the tests or if we could cut it down to only what's used?

Reply:
I think it is, actually. Those tests are all about querying deeply nested relationships and we need all the models here.